src/lang/en.json

```
@@ -514,6 +516,7 @@
     "passwordNotMatchMsg": "The repeat password does not match.",
     "notificationDescription": "Notifications must be assigned to a monitor to function.",
     "keywordDescription": "Search keyword in plain HTML or JSON response. The search is case-sensitive.",
+    "jsonQueryDescription": "Do a json Query against the response and check for expected value (Return value will get converted into string for comparison). check out <a href='https://jsonata.org/'>here</a> for the documentation about the query language, also <a href='https://try.jsonata.org/'>here</a> you can find a nice playground",
```

I am currently on my phone and cannot look at the rest of the codebase.
What I don't like about the current approach is the use of `v-html`.
Imo [`Component interpolation`](https://vue-i18n.intlify.dev/guide/advanced/component.html) is a better approach, but if the project uses `v-html`, this should be kept

```suggestion
    "jsonQueryDescription": "Do a json Query against the response and check for expected value (Return value will get converted into string for comparison). Check out <a href='https://jsonata.org/'>jsonata.org</a> for the documentation about the query language. A playground can be found <a href='https://try.jsonata.org/'>here</a>.",
```

---

I have seen that `v-html` is used in other parts of the project and I have used that too...